kubeadm/v1alpha3-config.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: kubeadm.k8s.io/v1alpha3
+kind: InitConfiguration
+apiEndpoint:
+  advertiseAddress: CONFIG_CLUSTER_PRIVATE_IP
+  bindPort: 6443
+nodeRegistration:
+  kubeletExtraArgs:
+    "feature-gates": "BlockVolume=true,CRIContainerLogRotation=true"
+
+
+---
+apiVersion: kubeadm.k8s.io/v1alpha3
+kind: ClusterConfiguration
+kubernetesVersion: CONFIG_KUBERNETES_VERSION
+apiServerCertSANs:
+  - CONFIG_CLUSTER_PUBLIC_IP
+apiServerExtraArgs:
+  authorization-mode: Node,RBAC
+certificatesDir: /etc/kubernetes/pki
+clusterName: kubernetes
+imageRepository: k8s.gcr.io
+
+
+---
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+containerLogMaxFiles: 1
+containerLogMaxSize: 10Mi
```

Comment:
I would make the `containerLogMaxSize` a TF variable with default to `100Mi`

Reply:
sure - will do.